ouroboros-consensus/docs/Onboarding.md

```diff
@@ -146,6 +146,4 @@ The following artifacts influence and/or describe the Consensus implementation.
 
 See
 https://github.com/input-output-hk/ouroboros-network/ouroboros-consensus/docs/Contributing.md
-and as usual the
-https://github.com/input-output-hk/ouroboros-network/ouroboros-consensus/README.md
-file.
+for all of our advice about approaching this repository.
```

Comment:
IMHO removed this file for now (in this PR). If we gonna need it (see my other PR) we will add it.

Reply:
Let's leave it in until another PR rearranges/replaces that content. I like that it collates those all of those links and does so alongside a very high-level summary.

Reply:
👍